tests/models/decoder_only/language/test_hybrid.py

```diff
@@ -8,7 +8,8 @@
 
 from ...utils import check_outputs_equal
 
-MODELS = ["ai21labs/Jamba-tiny-dev"]
+# This test is for the hybrid models
+MODELS = ["ai21labs/Jamba-tiny-dev", "ibm-fms/Bamba-9B"]
```

Comment:
Reminder to change before landing
```suggestion
MODELS = ["ai21labs/Jamba-tiny-dev", "ibm-ai-platform/Bamba-9B"]
```